common.lic

```diff
@@ -282,4 +282,27 @@ module DRC
       .map { |_name, properties| properties['abbrev'] }
       .each { |abbrev| fput("release #{abbrev}") }
   end
+
+  def necro_harvest_check
+    if !@necro_store
```

Comment:
The `rummage` helper already converts it to an array, so you won't have to use `scan` either. Just count the results from the helper.